stable/grc/templates/cert-policy-clustermanagementaddon.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  name: cert-policy-controller
```

Comment:
Is this the addon name to be displayed on UI? If yes, I prefer we call  it certificate policy addon instead of controller